acdc/TLACDCExperiment.py

```diff
@@ -432,11 +439,6 @@ def setup_corrupted_cache(self):
         if self.corrupted_cache_cpu:
             self.global_cache.to("cpu", which_caches="second")
 
-        if self.use_pos_embed and not self.zero_ablation:
```

Comment:
The bug was that this "post-hoc" way of randomising positional embeddings messed with the correctness of the complete graph in a very similar way to the zero ablation bug